frontend/src/components/navbar/AccountButton.js

```diff
@@ -0,0 +1,34 @@
+"use client"
+
+import React, { useState } from 'react';
+import { useRouter } from 'next/navigation';
+import styles from './AccountButton.module.css';
+
+
+const avatarIconPath = "/navbar/icons/avatar.png" // path for the avatar icon.
+
+//takes in a props{username, darkTheme} 
+//username can be changed to be the actual user id. 
+
+
+const AccountButton = (props) => {
```

Comment:
I believe you can unpack the props using code like the following:
```
const AccountButton = ({ username, darkTheme }) => {
     const [displayName, setDisplayName] = useState(username);
     ...
```
This makes it easier to understand what a component needs to be passed. 
ChatGPT also agrees: https://chatgpt.com/share/6723c8f5-7e9c-8004-bac9-2329873d84b5